src/styles.css

```diff
@@ -176,6 +176,18 @@
   src: url("https://sunflower-land.com/testnet-assets/fonts/Secondary.ttf");
 }
 
+@font-face {
+  font-family: "Born2bSporty";
```

Comment:
Born2bSporty FS is licensed under the [SIL Open Font License (OFL)](http://scripts.sil.org/OFL)